lib/is.js

```diff
@@ -48,6 +48,15 @@ const buffer = function (val) {
   return val instanceof Buffer;
 };
 
+/**
+ * Is this value a Uint8Array or Uint8ClampedArray object?
+ * @private
+ */
+const uint8Array = function (val) {
+  // allow both since Uint8ClampedArray simply clamps the values between 0-255
+  return val instanceof Uint8Array || val instanceof Uint8ClampedArray;
```

Comment:
Node.js `Buffer` objects will always satisfy the `instanceof Uint8Array` check. Perhaps verify `data.constructor.name` instead?